### ISSUE TITLE
[config-presets] config-presets' `extends` seems not to work correctly

### ISSUE THREAD

**Issue by kagankan** [markuplint/markuplint#576]
- Markuplint version: `3.0.0-alpha.5`
- Parser lang: HTML 
- Node.js version: `16.16.0`
- OS: Windows

## Describe the bug

I used v3 as a trial.
I'm sorry if I misunderstood, some rules in `markuplint:recommended` (v3) and `markuplint:recommended-static-html` are not working.

### Sample invalid HTML (`sample.html`)

```html
<html>
<head>
  <title>Document</title>
</head>
<body>
  sample
</body>
```

- No `h1`
- No `DOCTYPE`
- No `lang` attr in `html`
- No `charset="UTF-8"`
- No end tag of `html`

### Case 1 (❌ Incorrect): markuplint:recommended in v3

Config:

```
module.exports = {
  extends: ["markuplint:recommended"], // v3
};
```

Expected:

- Error for `Require <h1>`
- Error for `Require doctype`
- Error for `Require <html lang>`
- Error for `Specify charset=UTF-8`

https://github.com/markuplint/markuplint/tree/v3/packages/%40markuplint/config-presets

Output:

```
<markuplint> passed C:\Users\xxx\sample.html
```

- No Error

### Case 2  (❌ Incorrect): markuplint:recommended-static-html in v3

Config:

```
module.exports = {
  extends: ["markuplint:recommended-static-html"], // v3
};
```

Expected:

- Error for `Require <h1>`
- Error for `Require doctype`
- Error for `Require <html lang>`
- Error for `Specify charset=UTF-8`
- Error for `No omit end-tag`

Output: 

```
<markuplint> warning: 終了タグがありません (end-tag) C:\Users\xxx\sample.html:1:1
  1: <html>
  2: <head>
```

- Only  `No omit end-tag`

###  Case 3 (✅Correct): markuplint:a11y

Config:

```
module.exports = {
  extends: ["markuplint:a11y"], // v3
};
```

Output:

```
<markuplint> error: 要素「html」には属性「lang」が必要です (required-attr) C:\Users\xxx\sample.html:1:1
  1: <html>
  2: <head>
<markuplint> error: 要素「h1」が必要です (required-h1) C:\Users\xxx\sample.html:1:1
  1: <html>
  2: <head></head>
```

### Case 4 (✅ Correct): markuplint:recommended in v2

Config:

```
module.exports = {
  extends: ["markuplint:recommended"], // v2
};
```

Output: 

```
<markuplint> error: 文書型が必要です (doctype) C:\Users\xxx\sample.html:1:1
  1: <html>
  2: <head>
<markuplint> error: 要素「html」には属性「lang」が必要です (required-attr) C:\Users\xxx\sample.html:1:1
  1: <html>
  2: <head>
<markuplint> error: 要素「meta[charset="UTF-8" i]」が必要です (required-element) C:\Users\xxx\sample.html:2:1   
  1: <html>
  2: <head>
  3: ••<title>Document</title>
<markuplint> error: 要素「h1」が必要です (required-h1) C:\Users\xxx\sample.html:1:1
  1: <html>
  2: <head>
```

**Comment by kagankan** [comment]
I found the issue #554, after I subimitted!
The bug is already fixed, right?

**Comment by YusukeHirao** [comment]
@kagankan 

Oh, Sorry, I published `v3.0.0-alpha.6` now.
It includes #554 resolved.
Please check it.

**Comment by kagankan** [comment]
I confirmed it! Thank you. 😌
I'll close this issue.

However, `character-reference` errors appered (described in #577). 😢
